scripts/run_with_beaker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SOURCES_ARG="--source squad.latest:/squad --source glove.latest:/glove"
+RESULT_ARG="--result-path /output"
+GPU_ARG="--gpu-count=1"
+DETACH_ARG="--detach"  # or ""
+
+ECR_REPOSITORY=896129387501.dkr.ecr.us-west-2.amazonaws.com
+
+PARAM_FILE=$1
+EXPERIMENT_NAME=$2
+
+COMMIT=$(git rev-parse HEAD)
+IMAGE=$ECR_REPOSITORY/allennlp/allennlp-gpu:$COMMIT
```

Comment:
Yeah, I didn't copy that part of the code because modifying BiDAF config files makes the repo dirty, and I didn't want to have to make a commit just to run the experiment, especially because the docker image associated with an experiment _should_ be recoverable, and I can just inspect the code if I really want to know what was run.

As I said, this script is still a bit of a mess.  But I'm not sure I want to _require_ a clean repo to do this.  Maybe we can append a random number / string / hash to the commit id?